src/api/utils/fs_utils.ts

```diff
@@ -31,16 +31,24 @@ export function rimraf(thing: string) {
 }
 
 export function copy(from: string, to: string) {
-	if (!fs.existsSync(from)) return;
-
-	const stats = fs.statSync(from);
-
-	if (stats.isDirectory()) {
+	const copied = new Set();
+	const startDir = from + "/";
+	copyFiles(from, to);
+	return copied;
+  
+	function copyFiles(from: string, to: string) {
+	  if (!fs.existsSync(from)) return;
+  
+	  const stats = fs.statSync(from);
+  
+	  if (stats.isDirectory()) {
 		fs.readdirSync(from).forEach(file => {
-			copy(path.join(from, file), path.join(to, file));
+		  copyFiles(path.join(from, file), path.join(to, file));
```

Comment:
Your recursively calling a method, it would be better to call copy, instead of a method created within it.